src/mono/wasm/debugger/BrowserDebugProxy/Common/HelperExtensions.cs

```diff
@@ -23,10 +25,37 @@ public static string TruncateLogMessage(this string message)
                 ? message.Truncate(MaxLogMessageLineLength, ".. truncated")
                 : message;
 
-    public static void AddRange(this JArray arr, JArray addedArr)
+    public static void AddRange(this JArray arr, JArray addedArr, bool removeUnderscoreProperties = false)
     {
+        if (!removeUnderscoreProperties)
+        {
+            foreach (var item in addedArr)
+                arr.Add(item);
+            return;
+        }
         foreach (var item in addedArr)
-            arr.Add(item);
+            arr.Add(item.RemoveUnderscoreProperties());
+    }
+
+    public static JToken RemoveUnderscoreProperties(this JToken item)
+    {
+        return item.RemoveFields(new string[] {
```

Comment:
Do we really need a fixed list? We could remove all the fields with `"__"` prefix.